src/mount/mastercomm.h

```diff
@@ -40,6 +43,11 @@
 inline std::atomic_bool gIsDisconnectedFromMaster;
 #endif
 
+inline std::mutex acquiredFileMutex;
+// <inode, cnt> and sorted by inode
+using AcquiredFileMap = std::map<uint32_t, uint32_t>;
+inline AcquiredFileMap acquiredFiles;
```

Comment:
Consider #ifdef _WIN32 if they are only intended to be used in Windows.

Reply:
When rechecking this code, that portion is also used in the existing code before this proposed change, so it should be kept without the preprocessor directives.